ckan_cloud_operator/providers/db/proxy/azuresql/constants.py

```diff
@@ -0,0 +1 @@
+PROVIDER_ID='azuresql'
```

Comment:
the proxy can't be the same provider id as the main db provider

Reply:
`gcloudsql` and `rds` db providers has the same provider id as their proxies

Reply:
I guess that's also a potential bug, as they might overwrite one another's configuration... unless there's a hidden reason for this.

Reply:
done